support/reftest/reftest

```diff
@@ -196,8 +196,13 @@ def prepare(db, s3, config, opt):
             url, opt.key, opt.cert, opt.cacert
         )
 
-        # checksum verify
-        if cdn_data_checksum != item["sha256"]:
+        # unstable content check
+        is_unstable = False
+        if not item["sha256"]:
+            is_unstable = True
+
+        # For unstable content, it will skip the checksum verify
+        if not is_unstable and cdn_data_checksum != item["sha256"]:
```

Comment:
`if not is_unstable` means that there **is** an item["sha256"]
Was this meant to be `if is_unstable`?
If not, that case must be handled or upload_from_localfile will fail.

Reply:
@nathanegillett  When `is_unstable == True`, the `not is_unstable` will be False, it won't go to the if block inside.
For unstable content don't need a checksum-verify.

Reply:
Ah, I see now that item["sha256"] below was changed to cdn_data_checksum. Nevermind.

On another note, this seems a little overcomplicated. You just want to ensure there is an item["sha256"] value before checking cdn_data_checksum, right? Why not this;
```suggestion
        # For unstable content, it will skip the checksum verify
        if item["sha256"] and cdn_data_checksum != item["sha256"]:
```
`is_unstable` isn't used anywhere else so it doesn't really seem necessary.

Reply:
@nathanegillett done!